common/Helpers/RuntimeHelper.cs

```diff
@@ -34,4 +34,12 @@ public static bool IsCurrentProcessRunningAsAdmin()
         var identity = WindowsIdentity.GetCurrent();
         return identity.Owner?.IsWellKnown(WellKnownSidType.BuiltinAdministratorsSid) ?? false;
     }
+
+    public static void VerifyCurrentProcessRunningAsAdmin()

```

Comment:
I see that. This code is running in DevHome.PI.exe.... any thoughts on how we could best share code between these processes?

Reply:
I've created [`DevHome.Services.*` projects](https://github.com/microsoft/devhome/tree/main/services) which are application and UI technology agnostic, it's just bunch of dll services/library projects that anyone can use from anywhere. One way to leverage that, is adding to `DevHome.Services.Core` `IProcessInformation` or something like that would expose `IsRunningAsAdmin()` method. Then in Pi project, we just add `services.AddCore()` to register those services in Pi's container.

This can also be a follow up action, now that we've identified duplicate code. I'll leave it up to you 🙂

Reply:
Cool, I'll take it as a follow up. I suspect moving forward we'll want to share a bunch more code between the various components in DevHome.